var/spack/repos/builtin/packages/py-keras/package.py

```diff
@@ -9,17 +9,30 @@
 
 
 class PyKeras(PythonPackage):
-    """Deep Learning library for Python. Convnets, recurrent neural networks,
-    and more. Runs on Theano or TensorFlow."""
+    """Deep Learning for humans.
+
+    Keras is a deep learning API written in Python, running on top of the machine
+    learning platform TensorFlow. It was developed with a focus on enabling fast
+    experimentation. Being able to go from idea to result as fast as possible is
+    key to doing good research.
+    """
 
     homepage = "https://keras.io"
-    git = "https://github.com/keras-team/keras"
+    git = "https://github.com/keras-team/keras.git"
     url = 'https://github.com/keras-team/keras/archive/refs/tags/v2.7.0.tar.gz'
 
+    version('2.9.0', sha256='90226eaa0337573304f3e5ab44d4d9e3a65fe002776c5cbd0f65b738152c1084')
+    version('2.8.0', sha256='5e777b0101d8385d3a90fc9056f1b2f6313f2c830d2e8181828b300c9229ec0c')
     version('2.7.0', sha256='7502746467ab15184e2e267f13fbb2c3f33ba24f8e02a097d229ba376dabaa04')
     version('2.6.0', sha256='15586a3f3e1ed9182e6e0d4c0dbd052dfb7250e779ceb7e24f8839db5c63fcae')
-    version('2.5.0', commit='9c266106163390f173625c4e7b1ccb03ae145ffc')
+    version('2.5.0', commit='9c266106163390f173625c4e7b1ccb03ae145ffc', deprecated=True)
```

Comment:
Why is this one deprecated?

Reply:
There never was an official 2.5.0 release, just a release candidate. I guess I don't see a good reason to keep it.